index.html

```diff
@@ -0,0 +1,119 @@
+                    </div>
+                    <div class="section-skills-center"><img src="img/icons/photoshop.png"><br>
+                        <p>Photoshop</p>
+                    </div>
+                    <div class="section-skills-right"><img src="img/icons/html_css_js.png"><br>
+                        <p>Html, CSS, JavaScript</p>
+                    </div>
+                </div>
+        </section>
+        <section id="portfolio" class="section-Portfolio">
+            <p>Portfolio</p>
+            <div class="section-Portfolio-gallery">
+            <img src="img/plener/plener 01.jpg" alt="sesja ślubna w Bieszczadach" class="section-Portfolio-img">
+            <img src="img/plener/plener 02.jpg" alt="sesja ślubna w Karkonoszach" class="section-Portfolio-img">
+            <img src="img/plener/plener 03.jpg" alt="sesja ślubna w Tatrach" class="section-Portfolio-img">
+            <img src="img/plener/plener 04.jpg" alt="sesja ślubna w Lubuskim" class="section-Portfolio-img">
+            <img src="img/plener/plener 06.gif" alt="sesja ślubna ruchomy obrazek-gif Zielona Góra" class="section-Portfolio-img">
+            <img src="img/plener/plener 05.jpg" alt="sesja ślubna ruchomy obrazek-gif Zielona Góra" class="section-Portfolio-img">
+            </div>
+        </section>
+        <section id="contact" class="section-Contact-us">
+                <p>Contact us</p>
+                <div class="section-Contact-us-wrapper">
+                    <div class="section-Contact-us-wrapper-left">
+                        <div class="section-Contact-us-wrapper-left-email">
+                            <img src="img/icons/iconmonstr-email-6.svg" alt="icona email">
+                        </div>
+                        <div class="section-Contact-us-wrapper-left-text">
+                            <p>Please describe your question.<br>Send me a message and I will<br>answer you as soon as possible!</p>
+                        </div>
+                        <div class="section-Contact-us-wrapper-left-icon">
+                            <a class="first-icon" href="https://www.facebook.com/Wild-Plums-1108399012516313/?eid=ARBY9pW5RO9jVN0PUqi5BMsVEtmhv8_eXSDskRzl34O2K0zvN_Bh1O5VyosWQpbafolAv9Z5Sg3IB7LH" target="_blank">
+                                <img class="section-Contact-us-icon" src="img/icons/_ionicons_svg_logo-facebook.svg" alt="icon facebook"></a>
+                            <a href="https://www.instagram.com/wild_plums/?hl=pl" target="_blank">
+                                <img class="section-Contact-us-icon" src="img/icons/_ionicons_svg_logo-instagram.svg" alt="icon Instagram"></a>
+                            <a href="https://github.com/Bozik71" target="_blank">
+                                <img class="section-Contact-us-icon" src="img/icons/_ionicons_svg_logo-github.svg" alt="icon github"></a>
+                            <a href="https://www.linkedin.com/in/piotr-bożek-a3540015b/" target="_blank">
```

Comment:
++ za dodanie mediów społecznościowych z poprawnymi linkami

-- Na profilu LinkedIn masz literówkę :D 
`Fotografia to moje wieloletnie hobby, zainteresowanie, praca jednak kształcę się w programowaniu: **JavaScrip**, mySQL, HTML+CSS. `